### PR TITLE
Add 'gimel' symbol to dictionary.txt

### PR DESCRIPTION
**Pull request by yash2002109**
Signed-off-by: Yash Prakash <yash2002109@gmail.com>

### CODE REVIEW

**Comment by DenverCoder1** [comment]
Thanks for the contribution! Make sure to use tabs instead of spaces though, otherwise the dictionary won't load properly.

You can copy paste the tabs from other lines if it makes it easier.

If you have any questions feel free to ask

**Comment by yash2002109** [comment]
Could please check if the issue is rectified @DenverCoder1?

**Comment by DenverCoder1** [comment]
It looks there are 2 tabs instead of there being 1

image

File: dictionary.txt [hunk 1]
```diff
@@ -63,6 +63,7 @@
 \varkappa	ϰ	
 \varrho	ϱ	
 \epsilon	ϵ	
+\gimel	ג
```
Comment (DenverCoder1) [inline dictionary.txt]:
```suggestion
\gimel	ג	
```

**Comment by DenverCoder1** [comment]
Looks good :tada:

Thanks for contributing!

**Comment by yash2002109** [comment]
Thank you!

**Pull request merged**